### PR TITLE
Add OCPS commands and event.

### PR DESCRIPTION
**Pull request by ktlim**

### CODE REVIEW

File: sal_interfaces/OCPS/OCPS_Commands.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://project.lsst.org/ts/sal_objects/schema/SALCommandSet.xsl"?>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://project.lsst.org/ts/sal_objects/schema/SALCommandSet.xsd">
+  <SALCommand>
```
Comment (mareuter) [inline sal_interfaces/OCPS/OCPS_Commands.xml:4]: Only Subsystem, EFDB_Topic and Description are necessary for the command topic. Everything else can be removed.

File: sal_interfaces/OCPS/OCPS_Events.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://project.lsst.org/ts/sal_objects/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://project.lsst.org/ts/sal_objects/schema/SALEventSet.xsd">
+  <SALEvent>
+    <Subsystem>OCPS</Subsystem>
+    <Version>1.0.0</Version>
+    <Author>Kian-Tat Lim (ktl@slac.stanford.edu)</Author>
+    <EFDB_Topic>OCPS_logevent_job_result</EFDB_Topic>
+    <Alias>job_result</Alias>
+    <Explanation>https://dmtn-133.lsst.io/</Explanation>
```
Comment (mareuter) [inline sal_interfaces/OCPS/OCPS_Events.xml:10]: This is also going to go away at some point, so you can choose to do that now or wait until later.

**Review by mareuter — APPROVED** [review]
Small cleanup suggested, but otherwise this looks good to me.

**Comment by ktlim** [comment]
Thank you.  I updated the topics.  Please merge when ready.

**Comment by ktlim** [comment]
Never mind...  @rbovill gave me access, so I can merge myself!

**Pull request merged**